duckduckgo/iOS
# AppTP Onboarding, FAQ, Copy Updates

Task/Issue URL: https://app.asana.com/0/715106103902962/1204714133207123/f
Tech Design URL:
CC: @samsymons 

**Description**:
This PR introduces the onboarding flow for AppTP as well as the AppTP screen and beta copy updates.



**Steps to test this PR**:
1. On a fresh install tapping AppTP in settings should take you through the onboarding flow
2. The final page should prompt for VPN permissions and enable the VPN
3. The FAQ button in the activity view and the "Learn More" button in onboarding should open the FAQ page
4. The "What Are App Trackers" button in the activity view should open said article
5. Refer to asana task to verify copy changes



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
